src/irs/adapters/inbound/s3.py

```diff
@@ -47,87 +48,104 @@ def calc_part_ranges(
     return part_ranges
 
 
-async def get_download_url(*, object_id: str, bucket_id: str) -> str:
-    """Get object download URL from S3 inbox bucket"""
-    storage = get_objectstorage()
-    return await storage.get_object_download_url(
-        bucket_id=bucket_id, object_id=object_id
-    )
+@dataclass
+class S3IDs:
```

Comment:
Just a minor thing, but if we want to strictly follow PascalCase, it would be:
```suggestion
class S3Ids:
```

Reply:
Note however, if ID would be an acronym it should be all caps. PEP 8: "When using acronyms in CapWords, capitalize all the letters of the acronym. Thus HTTPServerError is better than HttpServerError."

But I would argue that "ID" is more like an abbreviation for "Identifier" than an acronym.

Not that we are splitting hairs here...

Reply:
Changed to lower case, no hair splitting needed ;)